satpy/enhancements/__init__.py

```diff
@@ -140,6 +140,75 @@ def func(band_data):
     return apply_enhancement(img.data, func)
 
 
+def reinhard(img, saturation=1.25, luminosity=1, **kwargs):
+    """Stretch method based on the Reinhard algorithm.
+
+    Luminosity has default value 1. Less is darker.
+    Saturation has default value 1.25. Less is grayer.
+
+    Reinhard, Erik, Michael Stark, Peter Shirley, and James Ferwerda:
+    ‘Photographic Tone Reproduction for Digital Images’, n.d., 10.
```

Comment:
Is there a DOI for this? I think we have a special "doi" role in the sphinx docs that would let you link to the article directly and show it in a nice way. I think we've used it for some of the DNB composites (ex. Zinke).

Reply:
Thanks